### PR TITLE
finish initial interface for code exercise #8

### PR DESCRIPTION
**Pull request by mistakeets**
Fair Warning: if you plan to download the folder and test the interface yourself, please note that I have not implemented a volume control yet and the oscillators / noise generates in the Web Audio API are quite loud by default. 

Currently working on exercise 8 (Play it again, JavaScript! Create an application that plays that Do–Re–Mi–Fa–So–La–Ti of the Solfège). The folder playMusic contains my basic idea for the interface, and I am still debating if I am going to use a slider on fixed steps to change between the musical notes/frequency to play Do - Re - Mi etc, or if I am going to use DOM selectors.

### CODE REVIEW

File: playItAgain/css/app.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
+body{
+  background-color:orange;
+  font-family: "Monospace";
+}
+
+div{
```
Comment (jaredatron) [inline playItAgain/css/app.css:6]: `div` is by far the most commonly used element. Putting padding on every `div` if going to hurt later. I would avoid this and just use a unique classname. `.padded-box` ?

File: playItAgain/css/app.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  padding: 20px;
+}
+
+#oscillator-list { /* Descendent Selector */
```
Comment (jaredatron) [inline playItAgain/css/app.css:10]: I never use ids. They are not more useful than classnames and they cause weird side affects you might not be aware of. just change these to classnames and you're good. unique classnames!!

File: playItAgain/css/app.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  outline: 2px solid #FFFFFF;
+}
+
+#solfege-list {
+  list-style-type: none;
+  padding-left: 0px;
+  color:white;
+}
+
+#do {
```
Comment (jaredatron) [inline playItAgain/css/app.css:46]: these names, ids or classes aside, are way way to short and therefore meaningless to other developers. Logger, more specific, more intuitively understandable names, are always better. Never choose short over understandable.

File: playItAgain/index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset = "UTF-8">
+    <title>Fun with WebAudio</title>
+    <script src='js/app.js'></script> 
+    <link rel="stylesheet" href="css/app.css">
+  </head>
+
+  <body>
```
Comment (jaredatron) [inline playItAgain/index.html:10]: the formatting in this HTML file is far from par. Its very difficult to read what elements are inside of what other elements. The `<div>` on line 12 should cause you to intent line 13. Please run this through a formatter.

File: playItAgain/index.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset = "UTF-8">
+    <title>Fun with WebAudio</title>
+    <script src='js/app.js'></script> 
+    <link rel="stylesheet" href="css/app.css">
```
Comment (jaredatron) [inline playItAgain/index.html:7]: `<link>` needs a closing tag or it needs to be self closing.

File: playItAgain/js/app.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
+"use strict"
+
+var audioContext = new AudioContext()
+window.onload = function() {
```
Comment (jaredatron) [inline playItAgain/js/app.js:4]: this is super oldschool. You should just register a DOM Ready event handler

```js
 document.addEventListener("DOMContentLoaded", function(event) {
    console.log("DOM fully loaded and parsed");
  });
```

File: playItAgain/js/app.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+      console.log("Oscillator is stopped. Waiting for oscillator to start")
+
+    }else{
+      freqSliderVal = document.getElementsByTagName("input")[1].value
+      osc.frequency.value = freqSliderVal;
+      console.log("Oscillator is playing. Frequence value is " + freqSliderVal)
+      osc.type = selectedWaveform
+    }
+  }, 50)
```
Comment (jaredatron) [inline playItAgain/js/app.js:58]: logging every 50 miliseconds is a lot of logging

**Comment by mistakeets** [comment]
Thanks for the feedback. After reviewing the code further I have decided to start over on this exercise which will re-factor everything into a function that plays back the musical scale.

**Pull request closed without merging**